### DISCUSSION TITLE
Regular 3D grains

### DISCUSSION THREAD

**Discussion by danishTUE** [neperfepx/neper#835] [Q&A]
Hi Romain,

I wanted to check with you if there is a way to generate a microstructure consisting of 3D regular array of idealized grains, let say of shape tetrakaidecahedron, using Neper?

Regards,

Danish

**Comment by rquey** [comment]
For this one, see `-morpho tocta`. There is also `-morpho cube`...

**Comment by danishTUE** [comment]
Hi Romain. I am interested in creating a multiscale tesselation by splitting truncated octahedron in four equal parts. What would be the best way to achieve this?
